source/MRViewer/MRUIStyle.h

```diff
@@ -45,6 +45,38 @@ struct ButtonIconCustomizationParams : public ButtonCustomizationParams
     bool flatBackgroundColor = false;
 };
 
+struct PlotAxis
+{
+    enum class TypeAxis
+    {
+        horizontal,
+        vertical,
+        none
+    } axisType = TypeAxis::none;
+    // the point from which the axes will be drawn
+    ImVec2 startAxisPoint;
+    // length dash without text
+    float lenDash = 8.0f;
+    // length dash with text
+    float lenDashWithText = 12.0f;
+    // text offset from dash
+    float textPadding = 3.0f;
+
+    // offset from the starting point along the axis
+    float offset = 0;
+    // the number of dash on the axis
+    size_t numDash = 5;
+    // the distance between the dash on the axis
+    float step = 0;
+    // what will be the first dash with the signature
+    size_t startTextPos = 0;
+    // sign every nth dash
+    size_t stepForText = 1;
+    // step for signature values
+    float axisStep = 1.0f;
```

Comment:
not clear

Reply:
how much to increase the value of the next dash